### PR TITLE
CodeQL: ignore dist folder

### PR DESCRIPTION
**Pull request by XhmikosR**
It probably makes sense since these are build files.

That being said, it might also make sense to add CodeQL to popper.js if they don't have it already. If someone could make an upstream PR that would be perfect :)

### CODE REVIEW

File: .github/codeql/codeql-config.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name: "CodeQL config"
+paths-ignore:
```
Comment (GeoSot) [inline .github/codeql/codeql-config.yml:2]: is it ok for now to keep this options inside the same job?

Comment (XhmikosR) [reply .github/codeql/codeql-config.yml:2]: What do you mean? If you mean not a separate file, I don't think that's possible.

Comment (GeoSot) [reply .github/codeql/codeql-config.yml:2]: Found this too.
https://github.blog/changelog/2022-08-31-code-scanning-customize-your-codeql-analysis-using-query-filters/


Anyway, it seems fine to go

**Review by GeoSot — APPROVED** [review]

File: .github/codeql/codeql-config.yml (hunk 1 above)
Comment (XhmikosR) [reply .github/codeql/codeql-config.yml:2]: Yeah, that link shows that it's not possible to exclude files/folders without a config file.

**Pull request merged**